---
title: AsposeArgumentException
second_title: Aspose.Email for .NET API 参考
description: 当提供给方法的参数之一无效时引发的异常
type: docs
weight: 260
url: /zh/net/aspose.email/asposeargumentexception/
---
## AsposeArgumentException class

当提供给方法的参数之一无效时引发的异常。

```csharp
public class AsposeArgumentException : AsposeException
```

## 构造函数

| 姓名 | 描述 |
| --- | --- |
| [AsposeArgumentException](asposeargumentexception#constructor)() | 初始化[`AsposeArgumentException`](../asposeargumentexception)类. |
| [AsposeArgumentException](asposeargumentexception#constructor_1)(Exception) | 初始化[`AsposeArgumentException`](../asposeargumentexception)类 引用了导致此异常的内部异常。 |
| [AsposeArgumentException](asposeargumentexception#constructor_2)(string) | 初始化[`AsposeArgumentException`](../asposeargumentexception)具有指定错误消息的类。 |
| [AsposeArgumentException](asposeargumentexception#constructor_3)(string, Exception) | 初始化[`AsposeArgumentException`](../asposeargumentexception)类 带有指定的错误消息和对导致此异常的内部异常的引用。 |
| [AsposeArgumentException](asposeargumentexception#constructor_4)(string, params object[]) | 初始化[`AsposeArgumentException`](../asposeargumentexception)具有指定错误消息的类。 |
| [AsposeArgumentException](asposeargumentexception#constructor_5)(string, string) | 初始化[`AsposeArgumentException`](../asposeargumentexception)具有指定错误消息的类。 |

## 特性

| 姓名 | 描述 |
| --- | --- |
| [ErrorDetails](../../aspose.email/asposeexception/errordetails) { get; } | 获取有关错误的额外信息 |
| override [Message](../../aspose.email/asposeexception/message) { get; } | 获取描述当前异常的消息。 |

## 方法

| 姓名 | 描述 |
| --- | --- |
| override [ToString](../../aspose.email/asposeexception/tostring)() | 返回代表当前对象的字符串。 |

### 也可以看看

* class [AsposeException](../asposeexception)
* 命名空间 [Aspose.Email](../../aspose.email)
* 部件 [Aspose.Email](../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
